QUICKSTART.md

```diff
@@ -207,10 +216,11 @@ HttpTextFormat.Getter<HttpExchange> getter =
 };
 ...
 public void handle(HttpExchange he) {
```

Comment:
Can we put a comment up above about where this `HttpExchange` class comes from?

Reply:
I linked `HttpExchange` to https://docs.oracle.com/javase/8/docs/jre/api/net/httpserver/spec/com/sun/net/httpserver/HttpExchange.html

Does that work for you?

Reply:
perfect (always scary to be using stuff in `com.sun`, but it's good to at least have the connection)